tests/block/k8s/create_block_test.go

```diff
@@ -57,7 +60,12 @@ func (s *K8sBlockImageCreateSuite) SetupSuite() {
 
 	var err error
 
-	s.testClient, err = clients.CreateTestClient(tests.Platform)
+	rook, err := installer.NewK8sRookhelper()
```

Comment:
i do like the pattern of each suite creating its own context, then they could have different settings for their environment if needed